modules/multi-arch-creating-podplacment-config.adoc

```diff
@@ -0,0 +1,52 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="multi-architecture-creating-podplacement-config_{context}"]
+
+= Creating a ClusterPodPlacementConfig custom resource
+
+After installing the Multiarch Tuning Operator, you must create a `ClusterPodPlacementConfig` custom resource (CR).
+
+The `ClusterPodPlacementConfig` CR is used to enable or disable the Multiarch Tuning Operator's pod placement operand.
+
+[discrete]
+== How to configure the ClusterPodPlacementConfig CR
+
+.Example `ClusterPodPlacementConfig` CR YAML file
+[source,yaml]
+----
+apiVersion: multiarch.openshift.io/v1beta1
+kind: ClusterPodPlacementConfig
+metadata:
+  name: cluster
+spec:
+  logVerbosityLevel: Normal <1>
+  namespaceSelector: <2>
+    matchExpressions:
+      - key: multiarch.openshift.io/exclude-pod-placement 
+        operator: DoesNotExist 
+----
+<1> Optional: You can set it to `Normal`, `Debug`, `Trace`, or `TraceAll`. It is set to `Normal` by default. 
+<2> Optional: You can configure the `namespaceSelector` to select the namespaces in which the Multiarch Tuning Operator's pod placement operand must process the `nodeAffinity` of the pods. All namespaces are considered by default.
+
+In this example, if the `multiarch.openshift.io/exclude-pod-placement` label is set in a namespace, the operand does not process the `nodeAffinity` of the pods in that namespace. Instead, the operand processes the `nodeAffinity` of the pods in namespaces that do not contain the label.
+
+If you want the operand to process the `nodeAffinity` of the pods only in specific namespaces, you can configure the `namespaceSelector` as follows:
+[source,yaml]
+----
+namespaceSelector:
+  matchExpressions:
+    - key: multiarch.openshift.io/include-pod-placement
```

Comment:
We have explained the configuration of the `namespaceSelector` field which contains all other sub fields. 
It contains the `label` and `operator` fields which must have different values based on the use case. 

hence, the use of footnotes is not suggested as it might be confusing to the readers.

We have explained the configuration using two different examples to clearly differentiate the two use cases.